api/src/pcapi/core/users/commands.py

```diff
@@ -64,3 +68,36 @@ def anonymize_inactive_users(category: str, force: bool) -> None:
         user_api.anonymize_non_pro_non_beneficiary_users(force=force)
     if category in ("pro", "all"):
         print("Anonymizing pro users X years after their last connection [not implemented]")
+
+
+# TODO (2024-02-19, cepehang) delete after usage
+@blueprint.cli.command("get_unknown_status_by_user_id")
+@click.option(
+    "--from-id",
+    type=int,
+    help="User id from which to check for unknown young status",
+    default=0,
+)
+def get_unknown_status_by_user_id(from_id: int) -> None:
+    last_id = from_id
+    unknown_status_by_user_id: dict[int, Any] = {}
+    while True:
+        users = User.query.filter(User.id > last_id).order_by(User.id).limit(1000).all()
+        if not users:
+            break
+
+        for user in users:
+            try:
+                young_status = get_user_subscription_state(user).young_status
+            except Exception:
+                unknown_status_by_user_id[user.id] = young_status
+            if not isinstance(young_status, YoungStatus):
+                unknown_status_by_user_id[user.id] = young_status
```

Comment:
pourquoi le if est hors du try ?

```suggestion
            try:
                young_status = get_user_subscription_state(user).young_status
	            if not isinstance(young_status, YoungStatus):
	                unknown_status_by_user_id[user.id] = young_status
            except Exception:
                unknown_status_by_user_id[user.id] = young_status
```

je ne comprends pas